mergify_engine/sub_utils.py

```diff
@@ -126,14 +126,3 @@ async def save_subscription_to_cache(installation_id, sub):
     r = await utils.get_aredis_for_cache()
     encrypted = _encrypt(sub)
     await r.setex("subscription-cache-%s" % installation_id, 3600, encrypted)
-
-
-def get_subscription(r, installation_id):
-    sub = _retrieve_subscription_from_cache(r, installation_id)
-    # TODO(sileht): We can remove the "tokens" check when we have flush
-    # the all tokens cache
-    if not sub or "tokens" not in sub:
```

Comment:
You're are going too far, the comment is just about `or "tokens" not in sub`

Reply:
🤦